.dockerignore

```diff
@@ -0,0 +1,12 @@
+tests
+Dockerfile
+README.md
+tox.ini
+.dockerignore
+.git
+.github
+.gitignore
+.idea
+.mypy_cache
+.pytest_cache
+.pre-commit-config.yaml
```

Comment:
❓ Probably not that important, but file seems to hit everything at the top level except for `src` and `LICENSE`. I'm assuming we'd like to include `LICENSE`?

Reply:
Also a good catch, no reason to keep that in the image so will add it here.